test/team_test.rb

```diff
@@ -5,13 +5,13 @@
 
 class TeamTest < Minitest::Test
   def setup
-    team_params = {
+    team_params = ({
```

Comment:
I'm curious about the addition of the parens here! My understanding was that adding them around a hash was only necessary if your test assertion expected value is a hash?

Reply:
Ruthie I think you're right, but it doesn't break the code so 🤷 ?